.buildkite/pipeline.yml

```
@@ -98,6 +98,21 @@ steps:
       - step: "tests"
         allow_failure: false
 
+  - label: ":argo: Update image tag for Fleet Server"
+    trigger: serverless-gitops-update-stack-image-tag
+    async: true
+    branches: main
+    build:
+      env:
+        IMAGE_TAG: "$DOCKER_IMAGE_TAG"
+        SERVICE: fleet-controller
+        NAMESPACE: fleet-ci
```

What does this namespace refer to ? Is it to kubernetes namespace or the docker namespace (URL) ? 

Should it be `observability-ci` ? 
```suggestion
        NAMESPACE: observability-ci
```

This is where docker images are being pushed right now by Jenkins: https://github.com/elastic/fleet-server/blob/main/.ci/Jenkinsfile#L19